config/oval.config

```diff
@@ -31,4 +31,4 @@ multi_platform_rhev =
 # OVAL Schema version
 #
 [Schema Version]
-oval_version = 5.10
+oval_version = 5.11
```

Comment:
No there is not.

I have brought this issue on the mailing list for wider awareness. https://lists.fedorahosted.org/pipermail/scap-security-guide/2015-April/006405.html